.github/workflows/issue-close-require.yml

```diff
@@ -0,0 +1,18 @@
+name: Issue Close Require
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  issue-close-require:
+    runs-on: ubuntu-latest
```

Comment:
We should limit the workflow to our repo (similarly the other one too):

```yml
if: github.repository == 'twbs/bootstrap'
```

Reply:
done